### ISSUE TITLE
Define contribution permissions and workflow

### ISSUE THREAD

**Issue by ChrisChinchilla** [ethereum/eth-wiki#41]
We need to define the permissions and workflow for contributors to edit and add pages, general idea at the moment is for the following:

- Allow anyone in the Ethereum github org to make changes
- Consider creating a new "wiki contributors" group that can also do the same for most engaged contributors
- Setup notifications / editorial inbox for all other changes and additions to pages
  - The person who made the change should be credited, no matter who edits/approves it.
- Anyone has previously made a contribution that was accepted can make changes without approval from then on, however and 'inbox' to see change notifications is still useful.

Once this is defined, document the process in #38

**Comment by ChrisChinchilla** [comment]
Yup, this kind of what I say above. In all honesty, I'm not sure if this setup will sync with what wiki.js offers and am weighing which platform (GH vs wiki.js) offers the best features we need right now. As everything is markdown under the hood switching platforms is pretty easy, and paths will all remain the same, so switching quickly between platforms if we find a better option is fairly straightforward, and won't change any external factors.

This was in progress today, updates soon.

**Comment by samajammin** [comment]
> The person who made the change should be credited

For this I suggest using [All Contributors](https://github.com/all-contributors/all-contributors) - we use it on [ethereum.org](https://github.com/ethereum/ethereum-org-website#contributors). It's a nice & easy way to show appreciation to contributors.

> In all honesty, I'm not sure if this setup will sync with what wiki.js offers

I disagree with this reasoning. Any project with a GH repo could follow that contribution process. It's completely agnostic of what the wiki tech stack is.

That said, I'm open to a discussion on potentially changing the wiki tech stack. I'd personally favor a Gatsby.js project given its MASSIVE community & plugin ecosystem vs. alternatives. Totally open to being persuaded otherwise & collaborating on a doc to decide potential pros & cons but I strongly feel this should not be the current priority. **We should first focus on completing the migration off the GH wiki**.

**Comment by MicahZoltu** [comment]
I think anyone should be able to submit a PR to update the content with a single button click (no login necessary).  Who gets auto-merged is a separate topic that I care far less about.  This assumes that edits will be done via GitHub long term?

**Comment by samajammin** [comment]
> Anyone can create content if they register.

@ChrisChinchilla I just tried to register an account but it didn't work. Here's the 

Browser console error:
```
```

Are you able to register an account?

**Comment by MicahZoltu** [comment]
I don't fully understand why I need to create an account, provide my email address, etc. to edit a page.

What I *want* is for the edit button to always be visible (no login needed) and when I click the edit button on https://eth.wiki/home it sends me to https://github.com/ethereum/eth-wiki/edit/master/home.md.  No login to eth.wiki necessary, no account necessary, etc.

For this workflow, I don't have to be a member of any organization, or receive pre-approval to edit, or anything like that.  I just need a GitHub account.

**Comment by jpitts** [comment]
@samajammin this error is due to lack of configuration for SMTP / DKIM, @wtf and I will add this then do a test to make sure it is set up properly.

**Comment by wtf** [comment]
@MicahZoltu We are gradually opening up the wiki for editing. Unsure if the front page should be editable but I agree that we should be able to log into the website irrespective of the email address - will look into this today.

Also note that the wiki has its own interface for editing, which is what the edit button should point to. One can always go to the GitHub repo and suggest edits. Maybe we should add a note to the wiki (along with a link to the repo) about this method @ChrisChinchilla?

**Comment by MicahZoltu** [comment]
@wtf What is the reason for having a separate editing interface/login/system from GitHub if you can *also* just edit on GitHub via pull request?  It *feels* unnecessary to have multiple systems for editing the wiki content.
